manifests/plugin/abrt/params.pp

```diff
@@ -1,6 +1,7 @@
 # Default parameters for the Pulp smart proxy plugin
 class foreman_proxy::plugin::abrt::params {
   $enabled                 = true
+  $version                 = 'installed'
```

Comment:
Interesting, that would work.  I wonder if it's not explicit enough though?  At least if it's defined here (or as I described) then you'd see the current value of the parameter with kafo..